lms/models/key.py

```diff
@@ -0,0 +1,41 @@
+import json
+from uuid import uuid4
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import UUID
+
+from lms.db import BASE
+from lms.models import CreatedUpdatedMixin
+
+
+class Key(CreatedUpdatedMixin, BASE):
+    __tablename__ = "key"
+
+    id = sa.Column(sa.Integer(), autoincrement=True, primary_key=True)
+
+    kid = sa.Column(UUID(as_uuid=True), default=uuid4, nullable=False)
+    """Unique identifier exposed to third parties"""
+
+    public_key = sa.Column(sa.Unicode)
+    """Public key a json JWK string"""
+
+    _private_key = sa.Column(sa.LargeBinary)
```

Comment:
This isn't private in a scoping sense, just in an RSA key pair sense?

```suggestion
    private_key = sa.Column(sa.LargeBinary)
```